index/install.php

```diff
@@ -216,7 +217,7 @@
 	$qtxt = "INSERT INTO regnskab (regnskab, dbhost, dbuser, db, version,bilag) values ";
 	$qtxt.= "('$db_navn' ,'$host', '$db_bruger', '$db_navn', '$version','0')";
 	db_modify($qtxt,__FILE__ . " linje " . __LINE__);
-	$qtxt = "CREATE TABLE online (session_id varchar(30), brugernavn text, db varchar(30), dbuser varchar(30), rettigheder varchar(30), ";
+	$qtxt = "CREATE TABLE online (session_id varchar(32), brugernavn text, db varchar(30), dbuser varchar(30), rettigheder varchar(30), ";
```

Comment:
This is good, however we need to also update the inbetween updates file such that all exsisting systems of saldi has this fix applied as well

Checkout [This code](https://github.com/DANOSOFT/saldi/blob/master/includes/betweenUpdates.php#L363-L369) where a similar fix was applied

Reply:
Thank for the link!
I have updated this PR.
However, this is not sufficient for existing installations as `betweenUpdates.php` are being included to late in `login.php`

The bug this PR adresses happens here at line 298:
https://github.com/DANOSOFT/saldi/blob/377e1dc8733ed9ec06c7fc96123890f3681a831d/index/login.php#L298

whereas `betweenUpdates.php` are not included until line 488:
https://github.com/DANOSOFT/saldi/blob/377e1dc8733ed9ec06c7fc96123890f3681a831d/index/login.php#L488

Any preferences for a way forward?
My best suggestion is to run a single "dirty in-between alteration" much earlier during login.

Reply:
Ah ofc. I think the best way forward would be to just modify login.php to include a lookup to the schema table to see if the column already has been updated, then update the column with the correct size.

We do this occasionally to help with migration in production, but we usually move them into between updates and later their concrete update file such as [this one](https://github.com/DANOSOFT/saldi/blob/master/includes/opdat_4.0.php)

So it is just important to add some notes as to why betweenupdates was not used, so it does not get "cleaned up" later in our development process

Reply:
Done